packages/cli/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@vercel/build-utils": "5.5.5",
+    "@vercel/error-utils": "1.0.0",
```

Comment:
This wont work if you plan to publish private. It would need to be `devDependencies`

Reply:
Unless we publish the cli with deps bundled or whatever then I'm making error-utils public so it can be required/imported correctly.

Reply:
Lets move it to `devDependencies` regardless because cli already bundles its dependencies.